spec/spec_helper.rb

```diff
@@ -77,7 +77,10 @@ def self.fake_matcher_description
 require 'rspec/support/spec'
 
 RSpec.configure do |config|
-  config.mock_with :rspec
+  config.mock_with :rspec do |mocks|
+    mocks.verify_partial_doubles = false
```

Comment:
Sadly enough, half of our own specs fail if this is on.

Reply:
Do you intend to change that? You seem to have started above?

Reply:
I've only fixed those examples that were also failing with `mocks.verify_partial_doubles = false` in `spec_helper` due to the use of isolated configuration using the default `true`.

Making all our `expect(Object.new).to receive(:foo)` verifiable is a huge task.